packages/release-mastermind/src/contexts/projects.ts

```diff
@@ -8,98 +1,215 @@
+import { Context } from "@actions/github/lib/context"
+import { LoggingDataClass, LoggingLevels } from "@videndum/utilities"
+import { Config, Runners, SharedConfig } from "../action"
+import { CurContext, ProjectContext, Version } from "../conditions"
+import { Utils } from "../utils"
+import { Contexts, log } from "./methods"
+import { Column } from "./methods/conventions"
+import { ProjectCreateBranch } from "./methods/createBranch"
+import { Milestones } from "./methods/handleMilestone"
+import { ExProjects } from "./methods/syncRemoteProject"
+
+/**
+ * The project configuration
+ */
+export interface ProjectConfig extends SharedConfig {
+	/**
+	 * Syncronise remote repository configuration.
+	 */
+	syncRemote?: ExProjects[]
+	/**
+	 * Open branch configuration
+	 */
+	openBranch?: ProjectCreateBranch
+	/**
+	 * Assign to milestone configuration
+	 */
+	assignMilestone?: {
+		[milestone: string]: Milestones
+	}
+}
+/**
+ * The project class.
+ * @private
+ */
 export class Project extends Contexts {
-  context: ProjectContext
-  config: ProjectConfig
-  constructor(
-    util: Utils,
-    runners: Runners,
-    configs: Config,
-    curContext: CurContext,
-    dryRun: boolean
-  ) {
-    if (curContext.type !== 'project')
-      throw new LoggingDataClass(LoggingLevels.error, 'Cannot construct without issue context')
-    super(util, runners, configs, curContext, dryRun)
-    this.context = curContext.context
-    if (!configs.project)
-      throw new LoggingDataClass(LoggingLevels.error, 'Cannot start without config')
-    this.config = configs.project
-  }
+	context: ProjectContext
+	config: ProjectConfig
+	constructor(
+		util: Utils,
+		runners: Runners,
+		configs: Config,
+		curContext: CurContext,
+		dryRun: boolean
+	) {
+		if (curContext.type !== "project")
+			throw new LoggingDataClass(
+				LoggingLevels.error,
+				"Cannot construct without project context"
+			)
+		super(util, runners, configs, curContext, dryRun)
+		this.context = curContext.context
+		if (!configs.project)
+			throw new LoggingDataClass(
+				LoggingLevels.error,
+				"Cannot start without config"
+			)
+		this.config = configs.project
+	}
+
+	/**
+	 * Parse the Project Context
+	 * @author IvanFon, TGTGamer, jbinda
+	 * @since 1.0.0
+	 */
+	static async parse(
+		utils: Utils,
+		config: Config,
+		context: Context
+	): Promise<ProjectContext | undefined> {
+		const project = context.payload.project_card
+		if (!project) {
+			return
+		}
+		log(
+			LoggingLevels.debug,
+			`context.payload.project_card: ${JSON.stringify(
+				context.payload.project_card
+			)}`
+		)
+
+		if (!project.content_url) throw new Error("No content information to get")
+		const issueNumber: number = project.content_url.split("/").pop()
+		const issue = await await utils.api.issues.get(issueNumber)
+
+		const labels = await utils.parsingData.labels(issue.labels).catch((err) => {
+			log(LoggingLevels.error, `Error thrown while parsing labels: `, err)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
      log(LoggingLevels.error, "Error thrown while parsing labels: ", err)
```